functions/model/organization/tark/removeOrgMember.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable max-len */
+/* eslint-disable object-curly-spacing */
+/* eslint-disable eol-last */
+
+/** *********************************************************
+ * Copyright (C) 2022
+ * Worktez
+ * Author: Aditya Khedekar <aditya3034@gmail.com>
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the MIT License
+ *
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the MIT License for more details.
+ ***********************************************************/
+
+const { updateApplication } = require("../../application/lib");
+const { updateMember, updateOrgRawData, getOrgRawData } = require("../../organization/lib");
+const { getAllTeams, updateTeamDetails } = require("../../teams/lib");
+const { getUserUseEmail, updateUser } = require("../../users/lib");
+exports.removeOrgMember = function(request, response) {
+  const orgDomain = request.body.data.OrganizationDomain;
+  const orgMembers = request.body.data.OrgMembers;
+  const remove = request.body.data.Remove;
+  let result;
+  let status = 200;
+  let flag = false;
+  const index = orgMembers.indexOf(remove);
+  if (index != -1) {
+    if (orgMembers.includes(remove)) {
+      flag = true;
+    }
+    orgMembers.splice(index, 1);
+    const updateJson = {
+      Active: false,
+    };
+    const p1 = updateMember(updateJson, orgDomain, remove);
+    if (!flag) {
+      getOrgRawData(orgDomain).then((rawData) => {
+        let totalNumberOfMembers = parseInt(rawData.TotalNumberOfMembers);
+        totalNumberOfMembers =totalNumberOfMembers- 1;
+        const updateRawDataInputJson = {
+          TotalNumberOfMembers: totalNumberOfMembers,
+        };
+        updateApplication(updateRawDataInputJson);
+        updateOrgRawData(updateRawDataInputJson, orgDomain);
```

Comment:
You cannot update the application rawdata with organization raw data.

Reply:
Shall I use another variable as inputJson to update the application rawdata